packages/api/src/external/carequality/command/cq-directory/create-or-update-cq-organization.ts

```diff
@@ -1,19 +1,19 @@
 import { CarequalityManagementAPI } from "@metriport/carequality-sdk";
+import { errorToString } from "@metriport/shared/common/error";
 import { makeCarequalityManagementAPI } from "../../api";
 import { CQOrganization } from "../../organization";
 import { CQOrgDetails } from "../../shared";
 
 const cq = makeCarequalityManagementAPI();
 
-export async function createOrUpdateCQOrganization(orgDetails: CQOrgDetails): Promise<void> {
+export async function createOrUpdateCQOrganization(orgDetails: CQOrgDetails): Promise<string> {
```

Comment:
Wanted to debug the response and found that we don't return the response, so added it here. We can use if we want, or not.